### PR TITLE
Cleanup of warnings and errors after inspecting project in IntelliJ

### PR DESCRIPTION
**Pull request by eluzgin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -242,9 +242,6 @@
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
 				<version>0.8.3</version>
-				<configuration>
-					<exclude>org.drools.*</exclude>
```
Comment (eluzgin) [inline pom.xml]: exclude doesn't belong here - fails validation

File: sm-core-model/pom.xml [hunk 2]
```diff
@@ -50,8 +50,6 @@
 		<commons-fileupload>1.3.3</commons-fileupload>
 		<commons-lang3.version>3.0</commons-lang3.version>
 		<commons-io.version>2.5</commons-io.version>
-		<jackson-version-databind>2.9.9.2</jackson-version-databind>
```
Comment (eluzgin) [inline sm-core-model/pom.xml]: removed duplicates inherited from parent pom.

File: sm-core-model/pom.xml [hunk 3]
```diff
@@ -110,12 +108,6 @@
 			<version>${jackson-version-databind}</version>
 		</dependency>
 
-		<dependency>
```
Comment (eluzgin) [inline sm-core-model/pom.xml]: duplicate declaration

File: sm-core-modules/pom.xml [hunk 4]
```diff
@@ -9,15 +9,6 @@
 		<artifactId>shopizer</artifactId>
 		<version>2.7.0</version>
 	</parent>
-
```
Comment (eluzgin) [inline sm-core-modules/pom.xml]: I see dead code ;-)

File: sm-core-modules/pom.xml [hunk 5]
```diff
@@ -67,8 +58,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<source>${java-version}</source>
```
Comment (eluzgin) [inline sm-core-modules/pom.xml]: Fixed typo - declared value with dot in parent pom.

**Comment by shopizer-ecommerce** [comment]
Thanks Eluzgin,

It will be merged today and retrofit to master (baseline to upcoming releases)

**Comment by eluzgin** [comment]
Hello,

I have been working on adding new crypto payment checkout method using
CoinPayments.
I have been unsuccessful adding new payment method into admin payments list.
My repo/branch: https://github.com/eluzgin/shopizer/tree/add-crypto-payments
Any tips appreciated!

I intend to contribute all code back to Shopizer project.

**Pull request closed without merging**